pages/guides/create-a-new-page.mdx

```diff
@@ -11,40 +11,43 @@ Allows you to add an empty accessible page to your application.
 <Steps>
 ### Create your Page component file
 
-Create a new file, `PageXXX`, in the appropriate module within the `features` folder.
+Create a new file, `PageMyNewScreen`, in the appropriate feature within the `features` folder (we will use an imaginary `feature-b` feature for this example).
 
-- **If it's the first page of a new module,** place it in a new folder named after the module.
-- **If you're adding a page to an existing module,** place it in that module's folder.
+- **If it's the first page of a new module,** place it in a new folder named after the feature (`feature-b` here).
+- **If you're adding a page to an existing feature,** place it in that feature's folder.
 
 <FileTree>
   <FileTree.Folder name="features" open>
-    <FileTree.Folder name="module-a" open />
-    <FileTree.Folder name="module-b" open>
-      <FileTree.File name="PageXXX" />
+    <FileTree.Folder name="feature-a" open />
+    <FileTree.Folder name="feature-b" open>
+      <FileTree.File name="PageMyNewScreen.tsx" />
+      <FileTree.File name="routes.ts" />
```

Comment:
This file is not yet explained, you ask to create it later, either mention we will explain later in the tutorial or remove it from the file tree, and add back a file tree where you ask to create the routes.ts file